CMakeLists.txt

```diff
@@ -613,7 +613,9 @@ if (${RELEASE_ID_SHORT} STREQUAL "Ubuntu")
   else (NOT ${RELEASE_NUM_SHORT} VERSION_LESS "19.0")
     set (LIBQRENCODE "libqrencode3 (>= 3.4.4-1)")
   endif (NOT ${RELEASE_NUM_SHORT} VERSION_LESS "19.0")
-  if (NOT ${RELEASE_NUM_SHORT} VERSION_LESS "20.0")
+  if (NOT ${RELEASE_NUM_SHORT} VERSION_LESS "22.10")
```

Comment:
I’m not too familiar with the cmake syntax, but it seems that previously the expression in parentheses was the same in the if and the else branches. Not sure why an else branch needs a condition or whether it should always match the condition in the first if.